ocfweb/stats/printing.py

```diff
@@ -0,0 +1,60 @@
+import pymysql
+from django.http import HttpResponse
+from django.shortcuts import render
+from matplotlib.figure import Figure
+
+from ocfweb.caching import periodic
+from ocfweb.stats.plot import plot_to_image_bytes
+
+
+# TODO: improve/move this after PyKota is gone
+CURRENT_SEMESTER_QUOTA = 100
+
+
+def pykota_connection(user='ocfpykota_user', password='not_a_secret'):
```

Comment:
as the password suggests, this is indeed not a secret password (it's read-only). it's already plaintext in puppet and any ocf user could read it straight out of pykota.conf.